README.rst

```diff
@@ -1,108 +1,42 @@
-Google Assistant Python Sample
-==============================
+Google Assistant SDK for Python
+===============================
 
-This repository contains a Python sample for the Google Assistant API.
+This package contains a collection of samples and tools to help you get started with the `Google Assistant SDK`_ using `Python`_.
```

Comment:
Agreed, but the reason I did this is because this file is what is displayed on `pypi` for the `google-assistant-sdk` package.

I will re-organize the repo to have a top-level README.md w/ an index like you suggested and keep the README.rst of the sdk focused on the Usage of the tool and samples it includes for people landing on the pypi package.

The sample README.rst (not shown in pypi, but linked from the docs) will show how to install, use and modify the gRPC sample outside of the -sdk package.